src/org/usfirst/frc/team930/robot/SpeedPID.java

```diff
@@ -0,0 +1,96 @@
+//package org.usfirst.frc.team930.robot;
```

Comment:
Lets get this out of this project, and into it's own. It is a very useful tool, but doesn't belong in this repository.